assets/src/components/propertiesPanel/minischedule.tsx

```diff
@@ -178,6 +200,7 @@ const Piece = ({
         <Row
           text="Start time"
           rightText={formattedScheduledTime(piece.start.time)}
+          belowText={startPlace}
```

Comment:
```suggestion
          belowText={piece.start.place}
```
this can be `piece.start.place` instead of checking the first trip's place. Then the variable can be removed.
(same for `piece.end.place` below.)